dali/sasha/saarch.cpp

```diff
@@ -210,6 +210,10 @@ void WUiterate(ISashaCommand *cmd, const char *mask)
                 return nullptr;
 
             name.setLength(name.length()-4);
+            //The .xml has been removed from the file name.
+            //If the name still contains a '.', the file is not a WU file.
+            if (strchr(name.str(), '.'))
+                return nullptr;
```

Comment:
I think this okay, given the other rough checks done here,
but really the whole of this func. could do with rewriting to parse the expected format a little more cleanly.

But also, it would be much better to stop these files getting as far as being use to call readWUIDFromFileName().
They are being returned from a IDirectoryIterator which uses a wildcard, which is causing a lot of unintended files to be picked up. It would be to ensure they were not returned from the iterator, by using a better wild card if possible.